target/linux/ipq40xx/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -0,0 +1,28 @@
+#!/bin/ash
+
+[ "$ACTION" == "add" ] || exit 0
+
+PHYNBR=${DEVPATH##*/phy}
+
+[ -n $PHYNBR ] || exit 0
+
+. /lib/functions.sh
+. /lib/functions/system.sh
+
+board=$(board_name)
+
+case "$board" in
```

Comment:
drop this file and patch the wireless macs via https://github.com/openwrt/openwrt/blob/master/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata.

Have a look at the meraki,mr33. It should work the same way.